.github/workflows/_elixir.yml

```diff
@@ -65,10 +65,10 @@ jobs:
         id: plt_cache
         with:
           path: elixir/priv/plts
-          key: dialyzer-ubuntu-22.04-${{ runner.arch }}-${{ steps.setup-beam.outputs.elixir-version }}-${{ steps.setup-beam.outputs.erlang-version }}-${{ hashFiles('elixir/mix.lock') }}
```

Comment:
For example, it detected that the `setup-beam` action did not have an output named `erlang-version`.